velox/functions/sparksql/RegisterArithmetic.cpp

```diff
@@ -41,7 +41,8 @@ void registerArithmeticFunctions(const std::string& prefix) {
   registerFunction<Log1pFunction, double, double>({prefix + "log1p"});
   registerFunction<ToBinaryStringFunction, Varchar, int64_t>({prefix + "bin"});
   registerFunction<ExpFunction, double, double>({prefix + "exp"});
-  registerBinaryIntegral<PModFunction>({prefix + "pmod"});
+  registerBinaryIntegral<PModIntFunction>({prefix + "pmod"});
+  registerBinaryFloatingPoint<PModFloatFunction>({prefix + "pmod"});
```

Comment:
Lets's update doc for this function, https://github.com/facebookincubator/velox/tree/main/velox/docs/functions/spark.

Reply:
Thanks, Just update doc for supported types.